### PR TITLE
Pair contactform

### PR DESCRIPTION
**Pull request by Kidd254**
in this branch I modified the html and the css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -271,6 +271,23 @@ <h1 class="header">I'm Lawrence Glad to see you</h1>
...
+            <form action="https://formspree.io/f/meqdkrwy" class="contact-form" method="post">
+                <input type="text" class="input-field" name="name" id="name" placeholder="Your Name" maxlength="30"
+                    required>
+                <input type="email" class="input-field" name="email" id="email" placeholder="Email" required>
+                <textarea name="message" class="text-area" id="message" cols="30" rows="10"
+                    placeholder="Write your message here" maxlength="500" required></textarea>
+                <input type="submit" class="submit-btn" value="Get in Touch">
+            </form>
+        </section>
 </body>
```
Comment (Nemwel-Boniface) [inline index.html]: - [ ] Great job working on your portfolio contact form. There is a white space that appears immediately below the contact form which is not supposed to be there according to the Figma design. Kindly ensure that you trim the white space an have the contact form as the last thing to meet all the project requirements. 

**This is how I am seeing the design right now:**

image

**This is the desired design:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Kidd254      :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on your portfolio so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Kidd254, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Professional PR. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [x] As the previous code reviewer mentioned, there is a whitespace at the bottom of your portfolio. This whitespace is due to a fixed height attribute that you added to a `.all` class that is applied to a `div` tag that is around the whole portfolio. Kindly remove this fixed height to make your portfolio fit on the mobile screen with a better UI.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] Your portfolio also has a `.background` class with a fixed width higher than any mobile screen width, making a horizontal scroll possible and a bad user experience. It would be nice if this fixed width is removed from that class. Fixed values can mess with the responsiveness of pages if not well applied.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Kidd254          :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes I requested good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**